packages/manager/modules/server-sidebar/src/controller.js

```diff
@@ -200,10 +205,15 @@ export default class OvhManagerServerSidebarController {
         } else {
           each(typesServices, (typeServices) => {
             this.addItems(get(parentService, 'children'), parent);
-
+            let items = get(typeServices, 'items');
             const hasSubItems = has(typeServices.type, 'types');
 
-            each(orderBy(typeServices.items, 'displayName'), (service) => {
+            if (!isEmpty(typeServices.type.filter)) {
+              // eslint-disable-next-line max-len
```

Comment:
Actually this comment could be removed.

To increase readability you could also do:

```js
items = typeServices.type.filter.fn(
  items,
  find(
    typesServices,
    service => get(service, 'type.category') === get(typeServices, 'type.filter.category'),
  ),
);
```